phpBB/language/en/email/test.txt

```diff
@@ -0,0 +1,9 @@
+Subject: phpBB is correctly configured to send emails
+
+Hello {USERNAME},
+
+Congratulations, if you received this email, phpBB is correctly configured to send emails.
+
+If you need any help, you can visit phpBB.com's community board - https://www.phpbb.com/community/
```

Comment:
In case you require assistance, please visit the phpBB support forums - https://www.phpbb.com/community/